plugin.video.massengeschmack/.gitignore

```diff
@@ -0,0 +1,27 @@
+# OS-specific files
```

Comment:
Wow, you guys really make it as hard as possible for plugin authors to match your standards.

When I merge my stuff into the pull request repository, I have to add all files to the index and squash them into a commit. Having at least a .gitignore makes sure I don't commit anything that shouldn't be there. Is it really so bad to have at least that in the repository?

Don't you guys do a git export from this repository anyways? Then all that stuff would be gone automatically.